src/app/friends/friends-list/friends-list.component.html

```diff
@@ -123,11 +110,11 @@ <h5 class="modal-title" id="staticBackdropLabel">Delete friend</h5>
       <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
     </div>
     <div class="modal-body">
-      Are you want to delete the {{deleteFriendName}}?
+      Are you want to delete the {{friendName}}?
```

Comment:
Message suggestion -> Do you want to delete the {{frientName}}?

Reply:
done